modules/snapshot/variables.tf

```diff
@@ -130,3 +131,10 @@ variable "eventbridge_schedule_expression" {
   nullable    = false
   default     = "rate(3 hours)"
 }
+
+variable "invoke_snapshot_on_start_enabled" {
+  description = "Toggle invocation of snapshot from Cloudformation. This can be useful for debug purposes if the lambda fails to complete successfully."
```

Comment:
```suggestion
  description = "Toggle invocation of initial resource snapshot via Lambda from Terraform."
```

Reply:
Aside from removing the CloudFormation reference, I think we should remove the debug sentence from both. It's the only data source for certain services. It's broadly useful to have immediately instead of waiting. Not just for debugging an issue (but especially).